content/TrueCommand/TCGettingStarted/Install/_index.md

```diff
@@ -141,15 +194,15 @@ docker run \--detach -v "/DockerDir:/data" -p 9004:80 -p 9005:443 ixsystems/true
 {{< hint info >}}
 Use [Windows compatible syntax](https://docs.microsoft.com/en-us/dotnet/standard/io/file-path-formats) when specifying paths in the Windows file system.
 For example, if the created directory for the TrueCommand image was created in the Windows <file>Documents</file> folder, the `docker` command would be:
-`docker run \--detach -v C:\Users\\Example\\Documents\\DockerDir`.
-In this command *C* is the drive letter, *Example* is the current user name, and *DockerDir* is the TrueCommand image directory.
+`docker run \--detach -v {C}:\Users\\{Example}\\Documents\\{DockerDir}`.
+`{C}` is the drive letter, `{Example}` is the current user name, and `{DockerDir}` is the TrueCommand image directory.
```

Comment:
same comment as above, are the names inside the { } brackets variables? use variable emphasis for the words if they are even if they appear in a command string with the CLI command emphasis also applied.
Typically you do the string with the variables emphasized then explain with:
Where *example* is the current user name, and *DockerDir is the TrueCommand image directory. I'll add an example of this in the style guide.